.releaserc

```diff
@@ -57,7 +57,7 @@
           }],
           ["@semantic-release/git", {
             "assets": ["./CHANGELOG.md", "./pom.xml", "./README.md"],
-            "message": "AWS Encryption SDK ${nextRelease.version} Release -- $(date +%Y-%m-%d) \n\n${nextRelease.notes}"
+            "message": "AWS Encryption SDK ${nextRelease.version} Release -- $(date +'%Y-%m-%d') \n\n${nextRelease.notes}"
```

Comment:
I am not confident this will work.
To be clear, it has never worked.

I do not think this tool allows arbitrary bash expressions.

We would need to read the tooling's user manual to determine how we can set a date parameter.

Reply:
From `semeantic-release/git` [README](https://github.com/semantic-release/git#message-examples), we can also use `${new Date().toISOString().slice(0, 10)}` to define the date parameter

Reply:
Try that. 
I bet `new Date().toISOString().slice(0, 10)` is JavaScript.
Not dead certain, but `semantic-release` is clearly a JS application.
It's certainly not a Bash expression.

Reply:
ack!